client/src/panels/panel_declarations/finances/drr_dp_resources/crso_by_prog.js

```diff
@@ -142,13 +196,30 @@ class PlannedProgramResources extends React.Component {
             </div>
           )}
           <div className="col-12 col-lg-8">
-            <WrappedNivoBar
-              data={data_by_year}
-              keys={Object.keys(graph_data)}
-              indexBy="year"
-              colors={(d) => colors(d.id)}
-              is_money={!is_fte}
-            />
+            <MediaQuery minWidth={minExtraLargeDevice}>
+              {get_nivo_bar_graph(34)}
+            </MediaQuery>
+            <MediaQuery
+              minWidth={minLargeDevice}
+              maxWidth={maxExtraLargeDevice}
+            >
+              {get_nivo_bar_graph(27)}
+            </MediaQuery>
+            <MediaQuery minWidth={minMediumDevice} maxWidth={maxLargeDevice}>
+              {get_nivo_bar_graph(31)}
+            </MediaQuery>
+            <MediaQuery minWidth={minSmallDevice} maxWidth={maxMediumDevice}>
+              {get_nivo_bar_graph(22)}
+            </MediaQuery>
+            <MediaQuery
+              minWidth={minExtraSmallDevice}
+              maxWidth={maxSmallDevice}
+            >
+              {get_nivo_bar_graph(17)}
+            </MediaQuery>
+            <MediaQuery maxWidth={maxExtraSmallDevice}>
+              {get_nivo_bar_graph(9)}
+            </MediaQuery>
```

Comment:
It works well enough, but all these hardcoded values are brittle against other nivo props/nivo internals (e.g. if they change the whole component's viewbox or something, whoch I don't trust them _not_ to do). Its a bit low value to make it more resilient to be fair, although if merged as is it would need a follow up issue to do something about some day (if nothing else, it will remind us to keep checking whether it's broken yet in the mean time, haha).

Reply:
Nivo always giving us pain😢